src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Windows.cs

```diff
@@ -25,8 +25,10 @@ public static string OSDescription
             {
                 if (null == s_osDescription)
                 {
-#if netcore50
+#if netcore50 || win8
                     s_osDescription = "Microsoft Windows";
```

Comment:
I first did netcore45 everywhere but hit an issue.  When parsing netcore45 nuget "conveniently" represents this as a framework object that ToString's to win8.  The result was a bad set of dependency groups in the nuspec because we didn't realize that there was indeed a win8 impl.  I could fix that up but it would require a round trip of buildtools.
I need to specify netcore45 for validation because that is the string we have in our validation targets and it needs to be an exact match.  Again could fix that but would require buildtools update.
I went with the smallest change here since I'm also trying to get this in escrow.
I do notice a few places where my comments were off, so I'll fix those up.